dapps/profile/src/components/buy-eve/buy-eve.ts

```diff
@@ -129,7 +129,7 @@ export class EvanBuyEveComponent extends AsyncComponent implements AfterViewInit
+  public isoCountries = [{"Code":"AD","Name":"Andorra"},{"Code":"AT","Name":"Austria"},{"Code":"BE","Name":"Belgium"},{"Code":"BG","Name":"Bulgaria"},{"Code":"HR","Name":"Croatia"},{"Code":"CY","Name":"Cyprus"},{"Code":"CZ","Name":"Czech Republic"},{"Code":"DK","Name":"Denmark"},{"Code":"EE","Name":"Estonia"},{"Code":"FI","Name":"Finland"},{"Code":"FR","Name":"France"},{"Code":"DE","Name":"Germany"},{"Code":"EL","Name":"Greece"},{"Code":"HU","Name":"Hungary"},{"Code":"IE","Name":"Ireland"},{"Code":"IT","Name":"Italy"},{"Code":"LV","Name":"Latvia"},{"Code":"LT","Name":"Lithuania"},{"Code":"LU","Name":"Luxembourg"},{"Code":"MT","Name":"Malta"},{"Code":"NL","Name":"Netherlands"},{"Code":"PL","Name":"Poland"},{"Code":"PT","Name":"Portugal"},{"Code":"RO","Name":"Romania"},{"Code":"SK","Name":"Slovakia"},{"Code":"SI","Name":"Slovenia"},{"Code":"ES","Name":"Spain"},{"Code":"SE","Name":"Sweden"},{"Code":"GB","Name":"United Kingdom"}];
```

Comment:
I think Andora shouldn't be in the list. UK may be removed soon ;)

Reply:
Andorra was included in the original list, clarified it with PGR, can be removed.
UK will be dealt with, when it is time... ^^